### PR TITLE
Added titles to pages

### PR DESCRIPTION
**Pull request by andylushman**
Added titles to pages. We will have to add titles to any pages that we create from now onward. For example, I will add a page tittle to the `sponsors` page when I create it.

### CODE REVIEW

File: app/views/site/program/tracks/show.html.slim [hunk 1]
```diff
@@ -1,2 +1,4 @@
+- content_for(:title, 'Programs')
```
Comment (scpaye506) [inline app/views/site/program/tracks/show.html.slim]: Should this title be `Tracks`?

File: app/views/articles/_article_form.html.slim [hunk 2]
```diff
@@ -1,4 +1,5 @@
 - content_for(:fullscreen_takeover, true)
+- content_for(:title, 'Submit Article')
```
Comment (scpaye506) [inline app/views/articles/_article_form.html.slim]: this is a partial, we should move the title to the `articles/new` and `articles/edit` pages. and `articles/show` probably needs something as well

File: app/views/site/program/tracks/show.html.slim (hunk 1 above)
Comment (andylushman) [reply app/views/site/program/tracks/show.html.slim]: Good catch!

File: app/views/articles/_article_form.html.slim (hunk 2 above)
Comment (andylushman) [reply app/views/articles/_article_form.html.slim]: good call. made titles for `Submit article`, `Edit Article` and on the `articles/show`...
```
- content_for(:title, 'Articles')
- content_for(:subtitle, "#{@article[:title]}")
```

**Pull request merged**